DatadogRUM/Sources/Debugging/RUMDebugging.swift

```diff
@@ -108,7 +109,7 @@ internal class RUMDebugging {
             canvas.addSubview(view)
         }
         if canvas.superview == nil,
-            let someWindow = UIApplication.managedShared?.keyWindow {
+           let someWindow = UIApplication.dd.managedShared?.keyWindow {
```

Comment:
💬 
`managedShared` is basically an instance of `UIApplication.shared`.
❓ 
Why don't we add extension to `UIApplication` directly? To me, it makes sense to access things via the `UIApplication.shared.foobar` which goes with existing APIs.

Reply:
This extension is now exposed publicly, we don't make an extension of `UIApplication` directly to prevent any confusion in customer's code completion. The reason for the `managedShared` is for compilation in app extensions, as stated in the comment:
```
/// `UIApplication.shared` does not compile in some environments (e.g. notification service app extension), resulting with:
/// _"shared' is unavailable in application extensions for iOS: Use view controller based solutions where appropriate instead"_.
///
/// As a workaround, this `managedShared` utility provides a key-path access to the `UIApplication.shared` to make the compiler pass.
```
👍

Reply:
I'm wondering if it would make sense to use `UIApplication.ddManagedShared` instead. The more specific the property, the less likely we have a conflict with developer's extension.

Reply:
Interesting issue, I kind of liked @maciejburda suggestion. If we are introducing this for the first time to customers ie no breaking change. It would work painlessly.